components/engine_test/src/lib.rs

```diff
@@ -282,6 +282,19 @@ pub mod kv {
         fn clone(&self) -> Box<dyn TabletFactory<KvTestEngine> + Send> {
```

Comment:
Interesting. In the prototype, a value local hashmap is used as cache to speed up access of tablet. That will make it !Sync. So `clone` is introduced to address the problem. Now that the implementation doesn't have a cache, I don't think `clone` is necessary anymore.

Reply:
We still have the local hashmap in TabletFactoryV2 and it's still Sync. But now we're using Arc<dyn TabletFactory> instead of Box<dyn TabletFactory>, so the customized clone() is not needed anymore.

Reply:
You mean `Arc<Mutex<HashMap<>>>`? Never thought accessing local cache would require lock..